docs_user/modules/proc_rolling-back-the-control-plane-adoption.adoc

```diff
@@ -2,27 +2,19 @@
 
 = Rolling back the control plane adoption
 
-If you encountered a problem during the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services that prevents you from completing the adoption procedure, you can roll back the control plane adoption.
+If you encountered a problem that prevents you from completing the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services, you can roll back the control plane adoption.
```

Comment:
Maybe: "If you encountered a problem with the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services, you can roll back the control plane adoption."

or

"If you are unable to complete the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services, you can roll back the control plane adoption."

or

"If you are unable to complete the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services, you can roll back the control plane adoption, so that you can troubleshoot and try again."